source/puppet/5.3/config_about_settings.markdown

```diff
@@ -174,4 +174,9 @@ On Windows 10, you can use PowerShell to set the system locale:
 
 ```
 Set-WinSystemLocale en-US
-```
\ No newline at end of file
+```
+
+{:.task}
+### Disabling internationalized strings
+
+Puppet 5.3.2 adds the optional Boolean `disable_i18n` setting, which you can configure in the `[main]` section of `puppet.conf`. If set to `true`, Puppet disables localized strings in log messages, reports, and parts of the command-line interface. This can improve performance when using Puppet modules, especially if [environment caching](./environments_creating.markdown#environment_timeout) is disabled, and even if you don't need localized strings or the modules aren't localized. This setting is `false` by default in open source Puppet.
```

Comment:
It can also be sit in `[user]`, `[agent]`, or `[master]`. People experiencing perf issues would primarily use it in `[master]`. People who want to force english messages would use `[main]` or `[user]`.

Reply:
Updated this section to be more specific, and removed specific sections from the release ntoes.